FirebaseInstallations/Source/Tests/Unit/FIRInstallationsAPIServiceTests.m

```diff
@@ -50,6 +53,10 @@ - (void)setUp {
   self.service = [[FIRInstallationsAPIService alloc] initWithURLSession:self.mockURLSession
                                                                  APIKey:self.APIKey
                                                               projectID:self.projectID];
+  NSString *const kHeartbeatStorageFile = @"HEARTBEAT_INFO_STORAGE";
+  GULHeartbeatDateStorage *dataStorage =
+      [[GULHeartbeatDateStorage alloc] initWithFileName:kHeartbeatStorageFile];
+  [[NSFileManager defaultManager] removeItemAtURL:[dataStorage fileURL] error:nil];
```

Comment:
The tests may be performed in parallel, so using a real heartbeat storage can interfere with other such tests and lead to flakes. We should mock FIRHeartbeatInfo class methods instead

Reply:
Yes makes sense. Updated